app/upgrades/v1/mainnet/token_distribution.go

```diff
@@ -1,26 +1,5 @@
 package mainnet
 
 const TokensDistributionStr = `{
-  "token_distribution": [
-    {
-      "address_sender": "bbn14d97wthm9fqvvdd96ax8lnfppwknndxztevs7k",
-      "address_receiver": "bbn13t5cnqj6t0p4xa40cwhmgv4wju0zl6g8slk8rz",
-      "amount": 100000
-    },
-    {
-      "address_sender": "bbn10d07y265gmmuvt4z0w9aw880jnsr700jduz5f2",
-      "address_receiver": "bbn1yl6hdjhmkf37639730gffanpzndzdpmhep8cg6",
-      "amount": 1500000
-    },
-    {
-      "address_sender": "bbn10d07y265gmmuvt4z0w9aw880jnsr700jduz5f2",
-      "address_receiver": "bbn1k6u5pge8w6lavtmunp02smehr4qtazkw8clg04",
-      "amount": 700000
-    },
-    {
-      "address_sender": "bbn10d07y265gmmuvt4z0w9aw880jnsr700jduz5f2",
-      "address_receiver": "bbn1dj2c57fjv6md7pzykh9y6h407ln6xxcw090hre",
-      "amount": 100000
-    }
-  ]
+  "token_distribution": []
```

Comment:
Why do we empty this?

Reply:
we are not distributing tokens with upgrade, but with a multisend